### DISCUSSION TITLE
GitHub UserID's for PO's as Vouching Person for Verification

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-infra#28] [General]
Hi all,

at the moment we are facing the problem that we need to verfiy new users for Catena-X NG

Problem here to mention the Vouching Person, this person (mostly PO's) need:
a GitHub Account
a Eclipse Foundation Account
and should be added to the contributor list of Eclipse-Tractusx

this would help us to identify and verify new users.

Additionally we can maybe generate a little list from it.

**Comment by Siegfriedk** [comment]
If its about catena-x ng, they can also send us their confluence/jira user perhaps?

**Reply by FaGru3n** [reply]
do we have a general List of all Po's in Confluence?

**Reply by almadigabor** [reply]
Is there a confluence page for who's who broken down by projects? That could help us. Also I beleive that maintainers of product teams in GH catenax-ng org should be feasable as vouching person.

**Reply by FaGru3n** [reply]
@almadigabor Cool Idea, because the maintainers are already responsibles, who will manage the access to the product repositories.

**Reply by FaGru3n** [reply]
added a bit more text in our issue template https://github.com/eclipse-tractusx/sig-infra/pull/45

**Comment by FaGru3n** [comment]
Infos already merged into our supporting issue-templates
